src/libprojplugin/libprojprojectmanager.cpp

```diff
@@ -25,7 +25,11 @@ ProjectExplorer::Project * OwnManager::openProject(const QString &Filename, QStr
             qWarning() << "Project file isn\'t a file" ;
             return nullptr;
     }
-    return new OwnProject(this, Filename);
+
+    QString contentOfProjectFile = readProjectFile(Filename);
+    if (contentOfProjectFile.isEmpty() || contentOfProjectFile.isNull())
+        return nullptr;
+    else return new OwnProject(this, Filename, contentOfProjectFile);
 }
 
 void OwnManager::registerProject(OwnProject * Project)
```

Comment:
#bug
Extract prefix "Own" from your classes
just put them into namespace libproject (for example) and name tmel just like Manager/Project/Node